pkg/common/httpclient.go

```diff
@@ -7,10 +7,19 @@ SPDX-License-Identifier: Apache-2.0
 package common
 
 import (
+	"crypto/tls"
 	"net/http"
 )
 
 // DefaultHTTPClient creates an http.Client with default parameters.
 func DefaultHTTPClient() *http.Client {
 	return http.DefaultClient
 }
+
+// InsecureHTTPClient creates an http.Client with disabled tls checks. Should be used only in tests.
+func InsecureHTTPClient() *http.Client {
+	//nolint:gosec
```

Comment:
What's the reason for disabling the linter here? If it's necessary, can we add a comment as to why?